setup/flavors/compose/docker-compose.yml

```diff
@@ -11,41 +12,39 @@ services:
     image: redis:alpine
     restart: always
     volumes:
-      - "$ROOT/redis:/data"
+      - "{{ root }}/redis:/data"
 
   # Core services
   front:
     image: mailu/nginx:{{ version }}
     restart: always
     env_file: {{ env }}
-    env:
-      - TLS_FLAVOR={{ tls_flavor or 'letsencrypt' }}
-      - ADMIN={{ expose_admin or 'no' }}
+    environment:
+      - TLS_FLAVOR={{ tls_flavor }}
+      - ADMIN={{ admin_enabled or 'false' }}
```

Comment:
@kaiyou these variables where mentioned here from the original template. Is there any good reason for doing so? I believe the should go back into the `mailu.env` file. We also will need those for all the other flavors.